core/src/ConfirmationModal.html

```diff
@@ -45,12 +45,7 @@ <h1 class="fd-modal__title">{content.header}</h1>
   $width: 32rem;
   .fd-modal {
     position: absolute;
-    box-sizing: border-box;
-    width: $width;
-    max-width: 90vw;
-    top: calc(#{$topNavHeight} + 1rem);
     left: calc(50% + #{$leftNavWidth}/ 2 - #{$width}/ 2);
```

Comment:
This should be removed as well if we want to have it in the middle of the screen.